test/cpp/api/modules.cpp

```diff
@@ -449,6 +449,32 @@ TEST_F(ModulesTest, L1Loss) {
   ASSERT_EQ(input.sizes(), input.grad().sizes());
 }
 
+TEST_F(ModulesTest, CosineSimilarity) {
+  CosineSimilarity cos(CosineSimilarityOptions().dim(1));
+  auto input1 = torch::randn({5, 3}, torch::requires_grad());
+  auto input2 = torch::randn({5, 3}, torch::requires_grad());
+  auto output = cos->forward(input1, input2);
+  auto s = output.sum();
+  s.backward();
+
+  ASSERT_EQ(output.ndimension(), 1);
+  ASSERT_EQ(output.size(0), 5);
+  ASSERT_EQ(input1.sizes(), input1.grad().sizes());
```

Comment:
Is there a way to check the value of `output` (e.g. by assigning special values to `input1` and `input2`)?